.github/workflows/withAptitude.yml

```diff
@@ -0,0 +1,110 @@
+  populate:
+    # Run on a single platform or use a matrix to test against several
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [ '3.8'  ]
+    env:
+       TMPTMPDIR: /tmp
+       OS_CHOICE: ${{ matrix.os }}
+       
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    name: gr-bokeh on ${{ matrix.os }} with python ${{ matrix.python-version }} 
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+       
+      - name: Make a place to store logs
+        run:  mkdir -p ~/artifacts
+
+      - name: Security first
+        run: |
+          mkdir ~/logs
+          sudo apt update
+          sudo apt install -y software-properties-common apt-utils 
+          sudo apt install -y unattended-upgrades
+          sudo unattended-upgrade -d >~/artifacts/securityFirst.log
+
+      - name: Ensure third-party apt support is installed
+        run: |
+          if [ "${OS_CHOICE}" = "ubuntu-latest" ]; then
+             sudo add-apt-repository -y ppa:gnuradio/gnuradio-releases-3.9
+          fi
+          sudo apt update
+
+      - name: Install the tangled web 
+        run: |
+          sudo apt install -qy build-essential npm nodejs cmake cmake-curses-gui ninja-build python3-pip
+          sudo apt install -qy lrzip cmake g++ libboost-all-dev libgmp-dev swig python3-numpy \
+                           python3-mako python3-sphinx python3-lxml doxygen libfftw3-dev \
+                           libsdl1.2-dev libgsl-dev libqwt-qt5-dev libqt5opengl5-dev python3-pyqt5 \
+                           liblog4cpp5-dev libzmq3-dev python3-yaml python3-click python3-click-plugins \
+                           python3-zmq python3-scipy python3-gi python3-gi-cairo gir1.2-gtk-3.0 \
+                           libcodec2-dev libgsm1-dev
+          sudo apt install -qy pybind11-dev python3-matplotlib libsndfile1-dev \
+                           python3-pip libsoapysdr-dev soapysdr-tools
+          pip install pygccxml
+          pip install pyqtgraph
+          sudo apt install -qy libiio-dev libad9361-dev libspdlog-dev python3-packaging
+
+      - name: Install gnuradio
+        run: |
+          sudo apt install -y gnuradio gnuradio-dev
+
+      - name: Install npm 
+        run: |
+          sudo apt install -y npm nodejs
+          echo sudo npm install -g npm@8.5 node@16
+          echo sudo chown -R 1000:1000 "/root/.npm"
+
+      - name: Checkout the current branch
+        uses: actions/checkout@master
+
+      - name: Checkout 2.3.1? of bokehjs
+        uses: actions/checkout@master
+        with:
+          repository: bokeh/bokeh
+          ref: refs/tags/2.3.1
+          path: 'build'
+          
+      - name: Install bokeh and bokehjs
+        run: |
+          pip install bokeh
```

Comment:
I've never had to do anything more than run `pip install bokeh` to install bokeh (and I've never had to explicitly add bokehjs). 
Is that required here because of the runner system? Or to ensure a specific version?

Reply:
I've had mixed results depending on os and architecture. Generally I install bokeh, bokeh info returns something reasonable, but either CMake says the library is not installed at buildtime, and/or the grcc-compiled tutorial.py reports that it cannot import bokehgui.